charts/prometheus-stackdriver-exporter/Chart.yaml

```
@@ -1,11 +1,11 @@
 apiVersion: v1
 description: Stackdriver exporter for Prometheus
 name: prometheus-stackdriver-exporter
-version: 1.12.0
+version: 1.13.0
```

please raise bugfix version instead of minor version
```suggestion
version: 1.12.1
```